src/Time.cpp

```diff
@@ -28,6 +30,14 @@ Time Time::now()
     return Time(static_cast<int64_t>(t.tv_sec) * UsecPerSec + t.tv_usec);
 }
 
+Time Time::nowUTC()
```

Comment:
This is actually wrong in most cases. gettimeofday does not return the local time, it returns the system time. Which on most default Linux distribution setups is already UTC. I personally assume that base::Time == UTC (because not doing so would be simply too full of corner-cases to account for)

The "culprit" that makes base::Time look like it stores local time is toString which breaks the time down to local time (using ... `localtime`). A non-localtime `toString` would be `gmtime`.

`fromString` and `fromTimeValues` are the real problem since `timegm`, the opposite of `gmtime`, is also a GNU extension. There are code snippets that should be usable to implement our own, but tbh I don't know how robust they actually are (and given how this stuff is complicated, I'd guess "not very")

Also, `timezone` is a non-standard GNU extension. I'd rather avoid using it with things like `musl` becoming rather common in the embedded world.

Again, I don't think this is worth our time *just* to have a proper fromString/fromTimeValues.